### PR TITLE
LPS-165984 Add IsEqualToIsAvailableForLongIntegerFields test

### PR DESCRIPTION
**Pull request by nkkwon**
Needs https://github.com/liferay-bpm-qa/liferay-portal/pull/384 pushed first for paths.
https://issues.liferay.com/browse/LPS-165984

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

**Comment by nkkwon** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 1]
```diff
@@ -2720,6 +2720,75 @@ definition {
...
+				key_fieldType = "Is Equal To",
+				locator1 = "ObjectField#TYPE_OPTION");
+
+			Type(
+				key_text = "Value",
+				locator1 = "TextInput#ANY",
+				value1 = "1");
+
+			ObjectCustomViews.clickSaveOnModal();
+		}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:2783]: This is fine for now but we should create a separate macro when creating the Custom Aggregation field.

**Review by timpak — APPROVED** [review]
LGTM!

**Comment by timpak** [comment]
Also, if we're going to add more Custom Aggregation field tests, we should explore extracting them into a separate .testcase file. ObjectFields.testcase is insanely bloated right now so we should look into splitting it into multiple .testcase files.

@andreefarias @evanilson-santana @Luizcarlosqueiroz FYI

**Comment by timpak** [comment]
ci:foward

**Comment by timpak** [comment]
ci:forward

**Pull request closed without merging**